spec/models/post_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  let(:user) do
+    User.new(
+      name: 'Tom',
+      photo: 'https://unsplash.com/photos/F_-0BxGuVvo',
+      bio: 'Teacher from Mexico.',
+      posts_counter: 1
+    )
+  end
+
+  let(:post) do
+    Post.new(
+      title: 'My first post',
+      text: 'This is my first post.',
+      author: user,
+      comments_counter: 1,
+      likes_counter: 2
+    )
+  end
+
+  it 'post should be valid with title, text and author' do
+    expect(post).to be_valid
+  end
+
+  it 'post should be invalid without title' do
```

Comment:
[ OPTIONAL] A better way to present the validation tests for the User and Post models
```
describe 'validations' do
  it { should validate_presence_of(:name) }
end
```
You can consider it for all your 6 validation tests 
![Screen Shot 2023-03-01 at 11 13 06 AM](https://user-images.githubusercontent.com/62849872/222240990-6ac7d7c8-4dfb-4633-9cfd-46238b1b8561.png)